app/views/taxYearOverview.scala.html

```diff
@@ -166,40 +185,46 @@ <h1 class="heading-xlarge">@heading</h1>
         <p class="heading-secondary">@messages("tax-year-overview.heading-secondary", (taxYear - 1).toString, taxYear.toString)</p>
     </header>
 
-
-    <dl class="govuk-summary-list">
-        <div class="govuk-summary-list__row">
-            <dd class="govuk-summary-list__value bold">
+    @if(overview.isDefined) {
```

Comment:
Optional suggestion: probably using the `overview.fold(...) {...}` here rather than `if` would allow a cleaner access to the content of the `Option`.
P.S. giving a name like `overviewOpt` in the view's argument list would allow to keep majority of existing code without changes, as the `overview` name can be used in the fold's branch for handling content of`Some`

Reply:
Fair point. Done